pages/register.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Form, Input, Button, Checkbox } from 'antd';
+import { UserOutlined, LockOutlined, MailOutlined } from '@ant-design/icons';
+import axios from '../axios';
+import Page from '../components/Page';
+
+export default () => {
+    const onFinish = (values) => {
+        const { email, firstname, password, lastname } = values;
```

Comment:
Accept email , username, name and password instead
Add a confirm password option too.